edvart/report_sections/timeseries_analysis/seasonal_decomposition.py

```diff
@@ -91,9 +90,11 @@ def seasonal_decomposition(
         """
         df = df.interpolate(method="time")
         if pd.infer_freq(df.index) is None and period is None:
-            warnings.warn(
-                "Period could not be inferred, please set the period parameter to a suitable value."
-                "Decomposition will not be plotted."
+            display(
+                Markdown(
+                    "Period could not be inferred, please set the `period` parameter"
+                    " to a suitable value. Not plotting seasonal decomposition."
+                )
```

Comment:
I'd make it look like an alert. E.g.:
```
                Markdown(
                    "<div class='alert alert-block alert-warning'>"
                    "Period could not be inferred, please set the `period` parameter"
                    " to a suitable value. Not plotting seasonal decomposition.</div>"
                )
```
which renders as:
![image](https://github.com/datamole-ai/edvart/assets/10757595/2d19f86c-5b1e-4119-b25f-a50ed01dc8e1)

Reply:
Good idea. I was actually thinking whether to keep this as a warning and this is a good middleground IMO.

Reply:
It would IMO make sense to display all warnings in this alert box. Or do you think that only some should be?

BTW: If used multiple times I'd create a util function `warning(text: str) -> Markdown` that would return this alert box.

Reply:
My logic here is that this is more of a warning, i.e. the user tried to display a plot but it could not be displayed because something is wrong (with the data or parameters).
The others are more of an info -- for example if a column has a lot of values, it makes no sense to display a histogram -- it's not something the user should fix.